draft-ietf-masque-connect-udp-listen.md

```diff
@@ -353,12 +353,14 @@ address, it MUST select an open port to bind to this request. From then and
 until the tunnel is closed, the proxy SHALL send packets received on these
 IP-port tuples to the client. The proxy MUST communicate the selected addresses
 and ports to the client using the "Proxy-Public-Address" header field. The
-header field is defined as a List of ip-port-tuples. The format of the tuple is
-defined using IP-literal, IPv4address, and port from {{Section 3.2 of
-!URI=RFC3986}}.
+header field is a List as defined in {{Section 3.1 of !STRUCTURED-FIELDS=RFC8941}}.
+Each member of the List is a Quoted String
+{{Section 3.2.4 of !INTERNET-MESSAGE-FORMAT=RFC5322}} comprised of the ip-port tuple . The
```

Comment:
Where did this reference come from? Why not use String from Section 3.3.3 of RFC 9651?